### PR TITLE
Feat/subheader cta legend

### PR DESCRIPTION
**Pull request by jbpenrath**
## Purpose
On funcampus & funcorporate platforms, we would like to display a custom message on the course detail page above the CTA in the subheader.

## Proposal
- [x] Override funcampus & funcorporate `course_detail` templates to add a message above subheader__cta
funcorporate result
funcampus result

### CODE REVIEW

File: sites/funcampus/src/backend/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -45,6 +45,10 @@ msgstr "Anglais"
 msgid "French"
 msgstr "Français"
 
+#: templates/courses/cms/course_detail.html:5
+msgid "Would you like to use this course with your students?"
+msgstr "Vous souhaitez utiliser ce cours avec vos étudiants?"
```
Comment (sampaccoud) [inline sites/funcampus/src/backend/locale/fr/LC_MESSAGES/django.po:50]:
```suggestion
msgstr "Vous souhaitez utiliser ce cours avec vos étudiants ?"
```

File: sites/funcorporate/src/backend/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -41,6 +41,10 @@ msgstr "Établissements"
 msgid "Persons"
 msgstr "Professeurs"
 
+#: templates/courses/cms/course_detail.html:5
+msgid "Would you like to offer this training to your employees?"
+msgstr "Vous souhaitez proposer cette formation à vos salariés?"
```
Comment (sampaccoud) [inline sites/funcorporate/src/backend/locale/fr/LC_MESSAGES/django.po:46]:
```suggestion
msgstr "Vous souhaitez proposer cette formation à vos salariés ?"
```

**Review by sampaccoud — APPROVED** [review]
A small neatpick on question marks that are preceded by a space in French only. Otherwise GTM!

File: sites/funcampus/src/backend/locale/fr/LC_MESSAGES/django.po (hunk 1 above)
Comment (jbpenrath) [reply sites/funcampus/src/backend/locale/fr/LC_MESSAGES/django.po]: Ok. I add a non breaking space then I'll merge!

**Pull request merged**